### PR TITLE
Add system-account privleges for addons

### PR DESCRIPTION
**Pull request by chuckbutler**
This adds a superuser level systemaccount (not ideal long term) to get
the addons working. This is a stop gap fix until
https://github.com/juju-solutions/bundle-canonical-kubernetes/issues/289
can be resolved.

### CODE REVIEW

File: cluster/juju/layers/kubernetes-master/reactive/kubernetes_master.py [hunk 1]
```diff
@@ -461,6 +461,17 @@ def addons_ready():
...
+    # the kube-system namespace. This gives the SA SU - this needs to be
+    # refactored to least privleged access.
+    try:
+        template_path = os.path.join(os.getenv('CHARM_DIR'), 'templates',
+                                     'kube-system-rbac.yaml')
+        check_call(['/snap/bin/kubectl', 'apply', '-f', template_path])
+    except CalledProcessError:
+        hookenv.log('Failed to apply kube-system rbac role for addons. '
+                    'Addons likely wont work.')
+
```
Comment (ktsakalozos) [inline cluster/juju/layers/kubernetes-master/reactive/kubernetes_master.py]: I think we are missing a return here so we do not move on with the cdk-addons.apply

File: cluster/juju/layers/kubernetes-master/templates/kube-system-rbac.yaml [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+metadata:
+  name: add-on-cluster-admin
+  namespace: kube-system
+subjects:
+- kind: ServiceAccount
+  name: default
+  namespace: kube-system
+roleRef:
+  kind: ClusterRole
+  name: cluster-admin
```
Comment (ktsakalozos) [inline cluster/juju/layers/kubernetes-master/templates/kube-system-rbac.yaml]: Do you think we should have "cluster-admin" as a constant in the .py file and render this file accordingly?

Comment (chuckbutler) [inline cluster/juju/layers/kubernetes-master/templates/kube-system-rbac.yaml]: I'm not sure I follow what you're asking here...

**Comment by ktsakalozos** [comment]
LGtM +1

**Pull request merged**